src/features/flows/thingy91x/index.ts

```diff
@@ -0,0 +1,297 @@
+    {
+        ref: 'nRF Cloud multi-service',
+        resources: [
+            {
+                app: 'pc-nrfconnect-serial-terminal',
+                description: 'Serial interface to send commands to the device.',
+                vComIndex: 0,
+                supplementaryLinks: [
+                    {
+                        label: 'Evaluate nRF Cloud services',
+                        href: 'https://docs.nrfcloud.com/GettingStarted.html#next-steps',
+                    },
+                ],
+            },
+        ],
+    },
+    {
+        ref: 'Shell Command Line Interface',
+        resources: [
+            {
+                app: 'pc-nrfconnect-serial-terminal',
+                description: 'Serial interface to send commands to the device.',
+                vComIndex: 0,
+                supplemetaryLinks: [
+                    {
+                        label: 'Modem shell commands',
+                        href: 'https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/samples/cellular/modem_shell/README.html#overview',
+                    },
+                    {
+                        label: 'AT Commands reference manual',
+                        href: 'https://docs.nordicsemi.com/bundle/ref_at_commands_nrf91x1/page/REF/at_commands/intro_nrf91x1.html',
+                    },
+                ],
+            },
+        ],
+    },
+];
+
+const learnConfig = [
+    {
+        label: 'Developer Academy',
+        description: 'Interactive online learning platform for Nordic devices.',
+        link: {
+            label: 'Nordic Developer Academy',
+            href: 'https://academy.nordicsemi.com/',
+        },
+    },
+    {
+        label: 'Best practices',
+        description:
+            'The main aspects and decisions you need to consider before and during the development phase of a low-power cellular Internet of Things product.',
+        link: {
+            label: 'nWP044 - Best practices for cellular IoT development',
+            href: 'https://docs.nordicsemi.com/bundle/nwp_044/page/WP/nwp_044/intro.html',
+        },
+    },
+];
+
+const developConfig = [
+    // Not in NCS
+    // {
+    //     ref: 'Start prototyping',
+    //     sampleSource: 'Unknown',
+    // },
+    {
+        ref: 'AT Commands',
+        sampleSource: 'nrf/applications/serial_lte_modem',
+    },
+    {
+        ref: 'nRF Cloud multi-service',
+        sampleSource: 'nrf/samples/cellular/nrf_cloud_multi_service',
+    },
+    {
+        ref: 'Shell Command Line Interface',
+        sampleSource: 'nrf/samples/cellular/modem_shell',
+    },
+];
+
+const appsConfig = [
+    'pc-nrfconnect-cellularmonitor',
+    'pc-nrfconnect-serial-terminal',
+    'pc-nrfconnect-programmer',
+    'pc-nrfconnect-ppk',
+];
+
+export default {
+    device: 'Nordic Thingy:91x',
+    flow: [
+        Info(infoConfig),
+        Rename(),
+        Program(programConfig),
+        Verify(verifyConfig),
+        SIM(),
+        Evaluate(evaluateConfig),
```

Comment:
In this step, you noticed, @boundlesscalm , that some links are missing from under Serial Terminal.